### PR TITLE
percona-server 8.0.13-3

### PR DESCRIPTION
**Pull request by teslamint**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

most changes copied from mysql 8.0.13 formula.

### CODE REVIEW

**Comment by fxcoudert** [comment]
We probably do not want the 5.7 versioned formula. It does not appear percona-server is in the top of our analytics, and no other formula in Homebrew depends on that specific version.

**Comment by fxcoudert** [comment]
@teslamint go ahead!

File: Formula/percona-server.rb [hunk 1]
```diff
@@ -42,16 +48,11 @@ def datadir
...
-    ENV.prepend "CPPFLAGS", "-DHAVE_MEMSET_S=1"
-
-    # https://dev.mysql.com/doc/refman/5.7/en/source-configuration-options.html
     # -DINSTALL_* are relative to `CMAKE_INSTALL_PREFIX` (`prefix`)
     args = %W[
       -DCOMPILATION_COMMENT=Homebrew
-      -DDEFAULT_CHARSET=utf8
-      -DDEFAULT_COLLATION=utf8_general_ci
+      -DDEFAULT_CHARSET=utf8mb4
+      -DDEFAULT_COLLATION=utf8mb4_general_ci
```
Comment (mbcoguno) [inline Formula/percona-server.rb]: Is `utf8mb4_0900_ai_ci` much better?

Comment (teslamint) [reply Formula/percona-server.rb]: @mbcoguno That is the default collation of MySQL 8.0, but current mysql formula doesn't. https://github.com/Homebrew/homebrew-core/commit/523589bf5b03053e40fb28681a5eac42655ce4cc#diff-4d75be177e3055025e7bef3144311559R49

File: Formula/percona-server.rb [hunk 2]
```diff
@@ -1,25 +1,24 @@
 class PerconaServer < Formula
   desc "Drop-in MySQL replacement"
   homepage "https://www.percona.com"
-  url "https://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.24-27/source/tarball/percona-server-5.7.24-27.tar.gz"
-  sha256 "999593afa241660bc7860264b46a826faf97d80d411966fade24084f47b249a0"
-
-  bottle do
```
Comment (fxcoudert) [inline Formula/percona-server.rb]: Do not touch the bottle block please.

File: Formula/percona-server.rb [hunk 3]
```diff
@@ -1,25 +1,24 @@
...
+  sha256 "ab00626775b85ea506c1d992d66acee50f0bfbde54b830731f61652e87ff93da"
 
   pour_bottle? do
     reason "The bottle needs a var/mysql datadir (yours is var/percona)."
     satisfy { datadir == var/"mysql" }
   end
 
   depends_on "cmake" => :build
+
+  # GCC is not supported either, so exclude for El Capitan.
```
Comment (fxcoudert) [inline Formula/percona-server.rb]: I'm not sure what you are trying to achieve with this line

File: Formula/percona-server.rb [hunk 4]
```diff
@@ -1,25 +1,24 @@
...
+  depends_on :macos => :sierra if DevelopmentTools.clang_build_version == 800
+
   # https://github.com/Homebrew/homebrew-core/issues/1475
   # Needs at least Clang 3.3, which shipped alongside Lion.
-  # Note: MySQL themselves don't support anything below El Capitan.
-  depends_on :macos => :lion
+  # Note: MySQL themselves don't support anything below Sierra.
+  depends_on :macos => :yosemite
+
```
Comment (fxcoudert) [inline Formula/percona-server.rb]: No extra blank line

File: Formula/percona-server.rb (hunk 1 above)
Comment (fxcoudert) [reply Formula/percona-server.rb]: We want to ship with the most common configuration.

**Review by fxcoudert — CHANGES_REQUESTED** [review]
Please squash everything into two commits:
- one that removes the versioned formula and the alias
- one that updates the main formula

File: Formula/percona-server.rb (hunk 3 above)
Comment (teslamint) [reply Formula/percona-server.rb]: That line introduced in this [commit](https://github.com/Homebrew/homebrew-core/pull/27210/commits/86c3e0af755f67f39e77f9f8943616b7e0411917#diff-4d75be177e3055025e7bef3144311559R38), and I just copied from mysql formula.
Should I remove this lines?

Comment (fxcoudert) [reply Formula/percona-server.rb]: Yes please

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
This needs rebasing on top of `master`.

**Comment by fxcoudert** [comment]
Thanks @teslamint

**Pull request closed without merging**